examples/serve/multi_node/multi_node.yaml

```diff
@@ -0,0 +1,49 @@
+# SkyServe YAML to launch a service where each replica contains multiple nodes.
+# Preemption of one node will cause the replica to be preempted and new replica to be launched.
+
+service:
+  readiness_probe: 
+    path: /v1/models
+    initial_delay_seconds: 6000
+  replicas: 1
+
+envs:
+  MODEL_NAME: meta-llama/Llama-2-70b-hf
+  HF_TOKEN: <your-huggingface-token> # TODO: Replace with huggingface token
+
+num_nodes: 2
+
+resources:
+  cloud: gcp
+  ports: 8000
+  accelerators: A100:2
+  use_spot: true
+  memory: 64+
+
+setup: |
+  conda create -n vllm python=3.11 -y
+  conda activate vllm
+  pip install -U "ray[default]"
+  pip install transformers
+  python -c "import huggingface_hub; huggingface_hub.login('${HF_TOKEN}')"
+
+  # Need to install vllm from source.
+  pip install git+https://github.com/vllm-project/vllm.git@563c1d7ec56aa0f9fdc28720f3517bf9297f5476
+
+
+run: |
+  conda activate vllm
+  head_ip=`echo "$SKYPILOT_NODE_IPS" | head -n1`
+  num_nodes=`echo "$SKYPILOT_NODE_IPS" | wc -l`
+  num_gpus=$(($SKYPILOT_NUM_GPUS_PER_NODE*$num_nodes))
+  if [ "${SKYPILOT_NODE_RANK}" == "0" ]; then
+      ray start --head --port=6379  --disable-usage-stats
```

Comment:
why do we need to manually ray start here? i suppose skypilot already created a cluster for us?

Reply:
also, just curious, does 6379 have some special meaning here?

Reply:
ray cluster or ray will not be visible in the conda environment. 
Following https://github.com/skypilot-org/skypilot/blob/master/examples/distributed_ray_train/ray_train.yaml#L26

Reply:
SG